testing/ncids-css-testing/stories/components/nci-autocomplete/nci-autocomplete.scss

```diff
@@ -0,0 +1,129 @@
+@import '@nciocpl/ncids-css/scss/lib/normalize';
+@import '@nciocpl/ncids-css/scss/packages/required';
+@import '@nciocpl/ncids-css/scss/packages/global';
+@import '@nciocpl/ncids-css/scss/components/nci-header/nci-extended';
+@import 'uswds/dist/scss/base/accessibility';
+
+// copied from search partial in header
+.nci-header-search {
```

Comment:
What is going on here?

Reply:
accessibility removed and made a dependency for autocomplete because it needs `usa-sr-only`